storage/src/block_meta_storage.rs

```diff
@@ -14,27 +16,33 @@ use crate::{BlockHeaderWithHash, StorageError};
 use crate::num_from_slice;
 use crate::persistent::{Decoder, default_table_options, Encoder, KeyValueSchema, KeyValueStoreWithSchema, PersistentStorage, SchemaError};
 use crate::persistent::database::{IteratorMode, IteratorWithSchema};
+use crate::predeccessor_storage::{PredecessorStorage, PredecessorKey, STORED_PREDECESSORS_SIZE};
 
 pub type BlockMetaStorageKV = dyn KeyValueStoreWithSchema<BlockMetaStorage> + Sync + Send;
 
 pub trait BlockMetaStorageReader: Sync + Send {
     fn get(&self, block_hash: &BlockHash) -> Result<Option<Meta>, StorageError>;
 
+    /// Returns n-th predecessor for block_hash
+    fn find_block_at_distance_old(&self, block_hash: BlockHash, distance: i32) -> Result<Option<BlockHash>, StorageError>;
```

Comment:
@adonagy 
Ado, remove this from trait, and just for tests implement `ind_block_at_distance_old` function in test module using `get` function